src/Build/BackEnd/BuildManager/BuildParameters.cs

```diff
@@ -725,6 +728,11 @@ public bool Interactive
             set => _interactive = value;
         }
 
+        public bool EnableGraphBuild
```

Comment:
I had to make it `public` since `XMake.cs` is in a different dll...

This is the flag future work will hook off of though.

Potentially though if graph builds "wrap" the build manager, then this flag should be renamed something more like "isolated project build" or something since it'll just enable the export enforcement stuff and the scheduling would all be done "outside" of this. If OTOH we have `BuildManager` take a `ProjectGraph`, then this flag is fine here, or perhaps should even not be settable and just be implied when a `ProjectGraph` is supplied.

I expect this to change a little as we continue development, which is why I'm not super happy this needs to be public...

Reply:
I think we can manage the API changes; it's definitely not bad enough to add friction to making forward progress here. Maybe we should consider adding a diff-API-against-last-release test to CI.